client/app/states/templates/editor/editor.state.js

```diff
@@ -0,0 +1,40 @@
+/* eslint camelcase: "off" */
```

Comment:
shouldn't need the eslint exception... or my :eyes: might be gettin' old :P :laughing: